doc/source/programs/gdal_vector_geom_simplify.rst

```diff
@@ -0,0 +1,71 @@
+.. _gdal_vector_geom_simplify_subsubcommand:
+
+================================================================================
+"gdal vector geom simplify" sub-sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Simplify geometries of a vector dataset.
+
+.. Index:: gdal vector geom simplify
+
+Synopsis
+--------
+
+.. program-output:: gdal vector geom simplify --help-doc
+
+Description
+-----------
+
+:program:`gdal vector geom simplify` simplify geometries ensuring that no
+segment is longer than the specified tolerance.
```

Comment:
New description for simplification :) but how to express it right?
Perhaps mention Douglas-Peucker, I guess that it is what is used, and the desciption could be something like 
"Simplifies a geometry by removing vertices. The tolerance value defines the maximum distance that is allowed between the original vertex and the simplified curve."

I think that GEOS supports also Visvalingam–Whyatt, but that can be reserved for the future.